src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java

```diff
@@ -70,8 +68,18 @@ public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper
                 caseName = name.getFullNameNoTitle();
             }
 
+            Benefit benefit = Benefit.getBenefitByCodeOrThrowException(appeal.getBenefitType().getCode());
+
+            DynamicListItem caseManagementCategory = new DynamicListItem(benefit.getShortName(), benefit.getDescription());
+            List<DynamicListItem> listItems = Arrays.asList(caseManagementCategory);
 
             return SscsCaseData.builder()
+                    .caseNameHmctsInternal(caseName)
+                    .caseNameHmctsRestricted(caseName)
+                    .caseNamePublic(caseName)
+                    .ogdType("DWP")
```

Comment:
Guess this is fine for now as only accepting DWP through SYA for now but not very future proof if they add SSCS5 submissions to SYA